lnrpc/lightning.proto

```diff
@@ -1388,6 +1388,13 @@ enum CommitmentType {
     */
     // TODO(roasbeef): need script enforce mirror type for the above as well?
```

Comment:
The TODO here breaks the correct generation of the swagger.json and pb.go files. The generated files pick up the todo as the documentation instead of the multi-line comment above.

Reply:
Great catch! Should probably combine the two comments, as the documentation generator probably just takes the "closest" comment to the enum value.